Makefile

```diff
@@ -10,10 +10,7 @@ MANINSTALLDIR=${PREFIX}/share/man
 GO_MD2MAN ?= /usr/bin/go-md2man
 
 ifeq ($(DEBUG), 1)
-  GOGCFLAGS += "-N -l"
-else
-  GOGCFLAGS ?= ""
-  DEBUG ?= 0
```

Comment:
> `skopeobuildimage make binary-local $(if $(DEBUG),DEBUG=$(DEBUG))`

Thanks, that’s more readable than I thought it would be, PR updated to use this.